recipes/ncbi-cxx-toolkit-public/26.0/conanfile.py

```diff
@@ -0,0 +1,1450 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanException
+from conans.errors import ConanInvalidConfiguration
+import os
+import shutil
+import subprocess
+import tempfile
+import sys
+import stat
+
+class NcbiCxxToolkit(ConanFile):
+    name = "ncbi-cxx-toolkit-public"
+    license = "CC0-1.0"
+    homepage = "https://ncbi.github.io/cxx-toolkit"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "NCBI C++ Toolkit"
+    topics = ("ncbi", "biotechnology", "bioinformatics", "genbank", "gene",
+              "genome", "genetic", "sequence", "alignment", "blast",
+              "biological", "toolkit", "c++")
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
```

Comment:
probaly fragile without `cmake_find_package` and/or pkg_config generators.

Reply:
"probably", "fragile" - what does it mean?

Reply:
It means that CMake configuration won't rely on Find or CMake config files of dependencies, which is the only robust way to find and inject dependencies.

Reply:
Looking at the build log, I have the feeling that your build system is managed by conan, which is not allowed because you can defeat conan-center policy.
https://github.com/ncbi/ncbi-cxx-toolkit-public/tree/release-26.0.1/src/build-system/cmake

For example I see that upstream CMakeLists call conan again with its own conanfile, it's forbidden.

Reply:
I do not know why you think so. It is possible to use the toolkit "with" conan. In this case, it does indeed "calls conan". But,  scenario of creating Conan-center package is different. In this case, the toolkit does not call conan - it does not need to. what it does though, it consumes information generated by conan (conanbuildinfo.cmake), nothing else

Reply:
Ok I see. So it doesn't call conan, but still relies on stuff defined in `cmake` generator?

Reply:
why not? in any case, it works with cmake only. will break? interesting. where can I read about the proposed changes?

Reply:
`cmake` generator will be removed in conan v2: https://docs.conan.io/en/latest/integrations/build_system/cmake.html

The new conan integration will be based on `CMakeToolchain` + `CMakeDeps`. Usually we prefer to see agnostic CMakeLists based on `find_package()`.